engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/CreateProcessInstanceTest.java

```diff
@@ -222,4 +227,63 @@ public void shouldCompleteUndefinedTask() {
             tuple(BpmnElementType.TASK, ProcessInstanceIntent.ELEMENT_COMPLETED),
             tuple(BpmnElementType.PROCESS, ProcessInstanceIntent.ELEMENT_COMPLETED));
   }
+
+  @Test
+  public void shouldActivateNoneStartEventWhenEventTriggerIsAvailableInState() {
```

Comment:
:wrench: We could add a comment with a link to the issue (e.g. "This is a regression test of ..."). 

The issue can give more context than the test case itself.